configs/snowflakeMapType.d.ts

```diff
@@ -50,4 +50,9 @@ export default interface snowflakeMap {
    * verified by mods)
    */
   Verified_Roles: string[];
+
+  /**
+   * A list of roles that members verified with /mod user Add Mature / Remove Mature should be Given / Removed
```

Comment:
'A list of roles that members [use a different word than verified to avoid confusion with the verification system] with /mod user add_mature/remove_mature should be given/removed'

Reply:
Fixed!